docs/content/latest/yugabyte-platform/create-deployments/create-universe-multi-zone.md

```diff
@@ -14,127 +12,143 @@ isTocNested: true
 showAsideToc: true
 ---
 
-This section will describe how to create a universe in one geographic region across multiple availability zones. We will examine the various nodes created by the Yugabyte Platform, run some workloads against this universe and take a look at the metrics against the running universe.
+  <ul class="nav nav-tabs-alt nav-tabs-yb">
 
-## 1. Create the universe
+<li>
+    <a href="/latest/yugabyte-platform/create-deployments/create-universe-multi-zone" class="nav-link active">
+      <i class="fas fa-building" aria-hidden="true"></i>
+Generic</a>
+  </li>
 
-If there are no universes created yet, the Dashboard page will look like the following. Click **Create Universe** to create the universe.
+  <li>
+    <a href="/latest/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes" class="nav-link">
+      <i class="fas fa-cubes" aria-hidden="true"></i>
+      Kubernetes
+    </a>
+  </li>
 
-![Dashboard with No Universes](/images/ee/no-univ-dashboard.png)
+</ul>
+
+This section describes how to create a YugabyteDB universe using any cloud provider, except Kubernetes, in one geographic region across multiple availability zones.
+
+## Create a universe
```

Comment:
I guess the Generic multi-zone doc is missing the prerequisites section. Good to have if we mention something like configure one of the VM-based cloud providers and then links of [AWS](https://docs.yugabyte.com/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws/), [GCP](https://docs.yugabyte.com/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/gcp/) & [Azure](https://docs.yugabyte.com/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/azure/).

Reply:
Added "Prerequisites" to the generic version.